scripts/pylib/twister/twisterlib/hardwaremap.py

```diff
@@ -287,11 +287,11 @@ def readlink(link):
         serial_devices = list_ports.comports()
         logger.info("Scanning connected hardware...")
         for d in serial_devices:
-            if d.manufacturer in self.manufacturer:
+            if d.manufacturer.upper() in [m.upper() for m in self.manufacturer]:
 
                 # TI XDS110 can have multiple serial devices for a single board
                 # assume endpoint 0 is the serial, skip all others
-                if d.manufacturer == 'Texas Instruments' and not d.location.endswith('0'):
+                if d.manufacturer.title() == 'Texas Instruments' and not d.location.endswith('0'):
```

Comment:
why do you need to do that?

Reply:
It is not needed to fix the reported bug, but I wanted to be consistent (case-insensitive) for all occurrences of d.manufacturer.

Reply:
please drop this, this makes it appear as if this a problem which is not.

Reply:
done